 lib/loadservice.js | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/lib/loadservice.js b/lib/loadservice.js
index fa4a136..41d3df9 100644
--- a/lib/loadservice.js
+++ b/lib/loadservice.js
@@ -216,7 +216,8 @@ exports.createWebinosJS = function(node_moduleDir, apiModules) {
         path.join(wrt_Dir,"webinos.session.js"),
         path.join(wrt_Dir,"webinos.js"),
         path.join(node_moduleDir,"webinos-api-serviceDiscovery","wrt","webinos.service.js"),
-        path.join(node_moduleDir,"webinos-api-serviceDiscovery","wrt","webinos.servicedisco.js")
+        path.join(node_moduleDir,"webinos-api-serviceDiscovery","wrt","webinos.servicedisco.js"),
+        path.join(node_moduleDir,"webinos-dashboard","wrt","webinos.dashboard.js")
     ];
 
     mandatoryWebinosJS.forEach(function(name){
